test/xUnit/csharp/test_DefaultVisit.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation.Language;
+using Xunit;
+
+namespace PSTests.Parallel
+{
+    public class MyCustomAstVisitor : ICustomAstVisitor2
```

Comment:
I suggest more correlated names:
```suggestion
    public class MyICustomAstVisitor2 : ICustomAstVisitor2
```

Reply:
Done.